lexicon.md

```diff
@@ -0,0 +1,110 @@
+
+### Primary Branch
+
+The main development branch in the [version control system](#version-control-system), representing the latest stable codebase. [Releases](#release) are typically made from this branch. Commonly named `main` or `master`. In some situations where branches are not featured, a [repository](#repository) with forked repositories will have the original repo acting as as an equivalent to the primary branch.
+
+### Project Documentation
+
+Written materials related to the project, such as user guides, developer guides, and contribution guidelines. These documents help users and developers understand, contribute to, and interact with the software. At release time, this may include [provenance] information, licensing details, and other metadata.
+
+### Provenance
+
+Information about the origin and history of the [released software assets](#released-software-assets). This may include details about its development, dependencies, vulnerabilities, contributors, and licensing.
+
+### Release
+
+- _(verb)_ The process of making a version-controlled bundle of assets available to users.
+- _(noun)_ A version-controlled bundle of code, [documentation](#project-documentation), and other assets made available to users. A release often includes release notes that describe the changes.
+
+### Released Software Assets
+
+Deliverables provided to users as part of a [release](#release). These assets can include binaries, libraries, or containers.
+
+### Releases
+
+Specific versions of the software that have been made available to users, typically accompanied by a [unique version identifier](#unique-version-identifier). Each release includes notes on updates and changes.
+
+### Repository
+
+A storage location managed by a [version control system](#version-control-system) where the project's [code](#commit), [documentation](#project-documentation), and other resources are stored. It tracks [changes](#changes), manages collaborator permissions, and includes configuration options such as branch protection and access controls.
+
+### Software Composition Analysis (SCA)
+
+The process of identifying and cataloging all components and dependencies in a software codebase. SCA is essential for managing security vulnerabilities and ensuring compliance with organizational policies.
+
+### Status Checks
+
+Automated tests or validations that run on [commits](#commit) before they are merged. Status checks ensure that any [changes](#changes) meet the project's quality and security standards.
+
+### Unique Version Identifier
+
+A label assigned to a specific [release](#release) of the software, such as `v1.2.3`. Commonly recommended formats are [Semantic Versioning](https://semver.org/) or [Calendar Versioning](https://calver.org).
+
+### Version Control System (VCS)
```

Comment:
I believe this needs to either not have *(VCS)* in the heading or the links need to be changed to use `#version-control-system-vcs`, similar to how was done for MFA above. (My weak preference is to drop *(VCS)*, but either one works.

Reply:
applied!